blivet/devices/btrfs.py

```diff
@@ -334,9 +333,9 @@ def _remove_subvolume(self, name):
 
     def list_subvolumes(self, snapshots_only=False):
         subvols = []
-        if flags.installer_mode:
+        if self.original_format.status:
```

Comment:
This was one place where I was uncertain whether the surrounding logic was really adequate to check, so yeah, I'm fine with using the flag instead.